Flicker/Global/Utils/FirebaseManager.swift

```diff
@@ -267,4 +267,56 @@ final class FirebaseManager: NSObject {
             print("error string Artist Model")
         }
     }
+    
+    func loadArtist(regions: [String]) async -> (artists: [Artist], cursor: QueryDocumentSnapshot?)? {
+        do {
+            var artists = [Artist]()
+            
+            var querySnapshot: QuerySnapshot
+            
+            if regions == ["전체"] {
+                querySnapshot = try await firestore.collection("artists").getDocuments()
+            } else {
+                querySnapshot = try await firestore.collection("artists").whereField("regions", arrayContainsAny: regions).getDocuments()
+            }
+            
+            querySnapshot.documents.forEach({ snapshot in
+                guard let artist = try? snapshot.data(as: Artist.self) else { return }
+                artists.append(artist)
+            })
+            
+            let cursor = querySnapshot.count < 5 ? nil : querySnapshot.documents.last
+            
+            return (artists, cursor)
+        } catch {
+            print("error to load Artist")
```

Comment:
p3

```suggestion
            print(error)
```

catch한 error를 출력할 수 있게 해주세용

Reply:
p3
5개로 제한한 이유가 궁금합니다!